src/overlays/actors/ovl_En_Giant/z_en_giant.c

```diff
@@ -352,20 +337,20 @@ void EnGiant_PlaySound(EnGiant* this) {
 
 void EnGiant_UpdatePosition(EnGiant* this, GlobalContext* globalCtx, u32 actionIndex) {
     CsCmdActorAction* actorAction = globalCtx->csCtx.npcActions[actionIndex];
-    f32 floatUnk10 = actorAction->unk10;
+    f32 startPosY = actorAction->unk10;
     s32 pad[2];
-    f32 floatUnk1C = actorAction->unk1C;
-    f32 functionTemp;
+    f32 endPosY = actorAction->unk1C;
+    f32 lerpScale;
```

Comment:
Well, I just called it that because `func_800F5A8C` is unnamed in this branch (it's named in the z_demo PR). The `lerp` part will be redundant once the function is named, though, so let me just change it to `scale`.